src/constants.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+
+RECIDIVIZ_DATA_URL = 'https://docs.google.com/spreadsheets/d/e/2PACX-1vTbxP67VHDHQt4xvpNmzbsXyT0pSh_b1Pn7aY5Ac089KKYnPDT6PpskMBMvhOX_PA08Zqkxt4zNn8_y/pub?gid=0&single=true&output=csv'
+MAPPINGS_URL = 'https://docs.google.com/spreadsheets/d/e/2PACX-1vT95DUfwcHbauuuMScd1Jb9u3vLCdfCcieXrRthNowoSbrmeWF3ibv06LkfcDxl1Vd97S5aujvnHdZX/pub?gid=1112897899&single=true&output=csv'
+
+OUTPUT_START_DATE = datetime.date(2020, 3, 31)
+
+OUTPUT_COLUMNS = ['Date',
+                  'Facility Type',
+                  'State',
+                  #'County',
```

Comment:
Is this here because it's meant to be optionally uncommented to enable more functionality or is it meant to be deleted before we merge?

Reply:
It's meant to capture the known headers in each source that we're specifically not using at this point. That's probably only worth noting for the question mark `Inmates Pending` column from `covidprisondata.com` though, so I've removed the other commented-out headers.